ArduCopter/mode.h

```diff
@@ -886,19 +895,21 @@ class ModeGuided : public Mode {
 
     // enum for GUID_OPTIONS parameter
     enum class Options : int32_t {
-        AllowArmingFromTX = (1U << 0),
+        AllowArmingFromTX   = (1U << 0),
         // this bit is still available, pilot yaw was mapped to bit 2 for symmetry with auto
-        IgnorePilotYaw    = (1U << 2),
+        IgnorePilotYaw      = (1U << 2),
         SetAttitudeTarget_ThrustAsThrust = (1U << 3),
+        DoNotStabilizePositionXY = (1U << 4),
+        DoNotStabilizeVelocityXY = (1U << 5),
     };
 
     void pos_control_start();
     void vel_control_start();
```

Comment:
to keep the _start and _run methods named similarly perhaps "vel_control_start()" should be renamed to "velaccel_control_start()"